### ISSUE TITLE
Please add support Jellyfin SyncPlay

### ISSUE THREAD

**Issue by gittes** [jellyfin/jellyfin-androidtv#538]
It would be awesome especially with the recent events if SyncPlay (introduced recently with Jellyfin 10.6.x) worked for the Jellyfin AndroidTV client. 

It would be also cool as a slight enhancement if the this implementation leveraged variable playback speeds to catchup when out of sync by only a few seconds or less instead of hard skipping and/or have the ability to define a offset tolerance before making an adjustment in seeking.

I'll let you guys figure out the interface design; would just love to have it.

It would just be nice to have SyncPlay on the big TV while using another device to chat or video conference with people instead of only watching SyncPlay groups in the web interface.

Some more superfluous suggestions: Perhaps existing SyncPlay groups could be presented as a channel/category/section similar to "Continue Playing" listed items with a "Now watching with others" section in the Home screen in addition to a group list browser. And there can be a "play with new SyncPlay" button next to the play button to initiate new SyncPlay group in the media item description screen.

**Comment by nielsvanvelzen** [comment]
We'd like to add SyncPlay too! It will take some time though. SyncPlay requires some changes in the apiclient which is undergoing a big refactor right now. After that's done we can start integrating SyncPlay.

**Comment by gittes** [comment]
> We'd like to add SyncPlay too! It will take some time though. SyncPlay requires some changes in the apiclient which is undergoing a big refactor right now. After that's done we can start integrating SyncPlay.

Fair enough and understood. I'll wait patiently and I am eager to see what the refactoring brings.

Thank you for setting expectations and the response.

**Comment by bqv** [comment]
What's the story folks?

**Comment by nielsvanvelzen** [comment]
Current state of the apiclient: waiting for 10.7 to release (minimum supported version) and then we can start integrating it into the app, that will take some time though.
When the integration is done we're rewriting all video/audio playback code and after that's done SyncPlay could be implemented. So it will take quite some time.

**Comment by GlassedSilver** [comment]
Which issues could I track to see how things progress? 
I searched for API as open issues, but there was nothing that seemed relevant. Maybe I should have searched differently?

**Comment by nielsvanvelzen** [comment]
The new apiclient (now called Jellyfin Kotlin SDK) is now in beta (since a few days). We will be testing it in the mobile android app before migrating the Android TV app.
SyncPlay is something that will be added as part of the playback rewrite (which is still in early stages and there is no issue for it).

**Comment by Siress** [comment]
Been a while. Any updates to this over the past year?

Loving the work everyone has done here... syncplay maybe the feature that pushes my wife over the edge of adoption since we use that around the house every day.

**Comment by nielsvanvelzen** [comment]
SyncPlay is a part of #1057, which is slowly progressing.

**Comment by JadeilsonSantos** [comment]
Good morning everybody! any news?

**Comment by AnimalLover2143** [comment]
this would really help lets say your at your computer and your family will be using the tv to watch a movie but groups isnt there but if it was you could watch with everyone.

**Comment by mels0n** [comment]
+1 - This would massively improve my user experience.

**Comment by gal1le0-cl** [comment]
Please this is so important

**Comment by dialupdisaster** [comment]
It's been a year since this was moved to started. Is there a PR somewhere? Can we contribute to it?

**Comment by tomerh2001** [comment]
I opened draft PR #5810 for native TV SyncPlay: creating/joining/leaving groups, shared playback and queue controls, and optional drift correction. It has 140 passing tests and playback checks between Jellyfin Web and an Android TV emulator. Physical TV testing is still needed.

There is an [unofficial preview APK](https://github.com/tomerh2001/jellyfin-androidtv/releases/tag/syncplay-preview-20260910) that installs alongside the Play Store app as Jellyfin Debug.

Disclosure: Codex wrote the implementation, PR description, and this comment at my request. I reviewed and accepted the code changes.
